examples/asset/utilization/AssetUtilization.cs

```diff
@@ -0,0 +1,47 @@
+﻿using NationalInstruments.SystemLink.Clients.AssetManagement;
+
+namespace NationalInstruments.SystemLink.Clients.Examples.Asset.Utilization
+{
+    /// <summary>
+    /// Example for the SystemLink Asset Management client API that
+    /// tracks utilization for assets present in the system.
+    /// 
+    /// Note: As a prerequisite for this example, ensure SystemLink Client is installed.
+    /// </summary>
+    class AssetUtilization
+    {
+        static void Main(string[] args)
+        {
+            /*
+             * The Asset Utilization Store enables asset utilization tracking. 
+             * It is created using a factory.
+             */
+            var utilizationStoreFactory = new AssetUtilizationStoreFactory();
+            var assetUtilizationStore = utilizationStoreFactory.CreateAssetUtilizationStore();
+
+            /*
+             * Tracking Asset Utilization can be configured to send heartbeats
+             * for ongoing utilization sessions, at predefined intervals. 
+             * By default, heartbeats are automatically sent every 5 minutes.
+             */
+            var utilizationConfiguration = StartUtilizationConfiguration.CreateDefault();
+
+            /*
+             * Start an utilization session for all assets in the system.
```

Comment:
"Start a utilization session..." This is one of the rare instances when you can use an "a" in front of a word that starts with a vowel because the word ("utilization") doesn't phonetically start with a vowel but a consonant (i.e. "yew-tilization"). Please update "an utilization" to "a utilization" throughout the doc.